packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -0,0 +1,54 @@
+﻿# Angular for XMCloud
+
+> Sitecore JSS Angular App for XM Cloud.
+
+[Documentation](<TODO>)
+
+This Single Page Application (SPA) built with Angular is designed to be fully compatible with XM Cloud, incorporating several key add-ons and features to streamline the development process and enable seamless integration. The supported key features are as follows:
+
+- Context ID / envoy proxy
+
+- Pages new editing integration via HTTP render engine endpoint (e.g. editing/render)
+
+- XMCloud proxy personalization
+
+- Forms support
+
+This SPA is tailored to enhance development workflows and enable full utilization of XM Cloud’s capabilities, providing a seamless and efficient foundation for developers.
+
+## Components and Supporting Applications
+
+The following components and supporting applications have been added to the Angular base app to ensure compatibility with XM Cloud:
+
+- `XMCloud Angular`: Adds support for the Sitecore Context data, which simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products. Also includes example components for SXA projects.
+
+- `XMCloud Proxy`: Adds integration with XMCloud for the JSS SPA applications. Considered experimental.
+
+## Environment Variables
+
+The following environment variables can be set to configure the angular app. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
+
+| Parameter                              | Description                                                                                                                                |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `PROXY_HOST`                        | Your XM Cloud Proxy hostname is needed to build the app and execute the client-side requests against the proxy server. Default value `http://localhost:3000`                                                                                                                  |
+| `PROXY_BUILD_PATH`                              | Your XM Cloud Proxy server path is needed to build the app. The build output will be copied to the XMCloud Proxy application path. Default value `..\\node-xmcloud-proxy\\dist`.
```

Comment:
default value here would depend on the name of the proxy folder. Something like <%xmcloud_proxy_path%>\\dist